sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -332,6 +332,11 @@ def test_upretty_modifiers():
     assert upretty( Symbol('x__dot') ) == u('x__dot')
 
 
+def test_pretty_Cycle():
+    from sympy.combinatorics.permutations import Cycle
+    assert pretty(Cycle(1, 2)) == '([1, 2],)'
```

Comment:
I believe ``srepr`` (in repr.py) is the printer producing a representation that can be used to reproduce the object (or possibly a copy). That is not what you get with
```
>>> srepr(Cycle(1, 2, 4)(5, 6))
'(1 2 4)(5 6)'
```
On the other hand, I think this is what  the "user friendly" printer in str.py should do. So it seems that the version of ``_print_Cycle`` currently in str.py should be moved to repr.py and replaced with a new printer.